src/js/utils/DOM.js

```diff
@@ -65,6 +65,17 @@ export const containsFocus = (node) => {
   return !!element;
 };
 
+export const withinDropPortal = (node) => {
```

Comment:
Since this is returning the drop portal id, what about calling this function `getDropPortalId`?

Reply:
This function changed around a bit in my latest commit. I left the name as `withinDropPortal` but I think with the changes this name now accurately reflects what the function is doing